modules/editable-layers/src/edit-modes/draw-polygon-mode.ts

```diff
@@ -13,155 +17,146 @@ import {
   GuideFeature,
   DoubleClickEvent
 } from './types';
-import {Polygon, FeatureCollection} from '../utils/geojson-types';
-import {getPickedEditHandle} from './utils';
+import {Position, FeatureCollection, Geometry} from '../utils/geojson-types';
 import {GeoJsonEditMode} from './geojson-edit-mode';
+import { ImmutableFeatureCollection } from './immutable-feature-collection';
+
 
 export class DrawPolygonMode extends GeoJsonEditMode {
+
+  holeSequence: Position[] = [];  
+  isDrawingHole = false;
+
   createTentativeFeature(props: ModeProps<FeatureCollection>): TentativeFeature {
-    const {lastPointerMoveEvent} = props;
+    const { lastPointerMoveEvent } = props;
```

Comment:
Nit: I'm seeing some inconsistencies. Does this repo have a linter like the core repo does?

Reply:
`yarn lint fix` before commit should run prettier with the .prettierrc in the repo